packages/jest-snapshot/src/index.ts

```diff
@@ -357,17 +357,12 @@ const _toMatchSnapshot = ({
       `${RECEIVED_COLOR('Received value')} ` +
       `${actual}`;
   } else {
-    expected = utils.removeExtraLineBreaks(expected);
-    actual = utils.removeExtraLineBreaks(actual);
-
     // Assign to local variable because of declaration let expected:
```

Comment:
Sounds like a workaround mentioned in the comment, can that be changed now?

Reply:
Oh yeah!